src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,132 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import * as Sentry from '@sentry/browser';
+
+import Button from 'app/components/button';
+import CommitRow from 'app/components/commitRow';
+import {Panel} from 'app/components/panels';
+import {promptsUpdate} from 'app/actionCreators/prompts';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+
+const SAMPLE_COMMIT = {
+  id: 'ff821e0',
+  author: {name: 'User Name'},
+  dateCreated: moment()
+    .subtract(3, 'day')
+    .format(),
+  repository: {
+    provider: {id: 'integrations:github', name: 'GitHub', status: 'active'},
+  },
+  message: 'ref(oops): This commit accidentally broke something',
+};
+
+class EventCauseEmpty extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+    organization: SentryTypes.Organization.isRequired,
+    project: SentryTypes.Project.isRequired,
+  };
+
+  state = {
+    shouldShow: undefined,
+  };
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    const {api, project, organization} = this.props;
+
+    api.request('/promptsactivity/', {
+      method: 'GET',
+      query: {
+        project_id: project.id,
+        organization_id: organization.id,
+        feature: 'suspect_commits',
+      },
+      success: (data, _, jqXHR) => {
+        this.setState({
+          shouldShow: this.shouldShow(data),
+        });
+      },
+      error: err => {
+        Sentry.captureException(err);
+      },
+    });
+  }
+
+  shouldShow({data} = {}) {
+    if (data && data.dismissed_ts) {
+      return false;
+    }
+    if (data && data.snoozed_ts) {
+      const now = Date.now() / 1000;
+      const snoozingTime = (now - data.snoozed_ts) / (60 * 24);
+      return snoozingTime > 3; // 3 days
+    }
+    return true;
+  }
+
+  handleClick(action) {
+    const {api, project, organization} = this.props;
+
+    const data = {
+      projectId: project.id,
+      organizationId: organization.id,
+      feature: 'suspect_commits',
+      status: action,
+    };
+    promptsUpdate(api, data).then(this.setState({shouldShow: false}));
+  }
+
+  render() {
+    const {shouldShow} = this.state;
+    if (!shouldShow) {
+      return null;
+    }
+
+    return (
+      <div className="box">
```

Comment:
@Chrissy I think we had something like this in mind

<img width="531" alt="Screen Shot 2019-07-08 at 12 54 55 PM" src="https://user-images.githubusercontent.com/16394317/60838482-a6a10600-a17f-11e9-8bcf-af25ad30d11c.png">

Reply:
this looks nice! My only suggestion is that the containing box is out of style and the border is too thick. I would check storybook.getsentry.net and see if we have any pre-made container components that fit the bill.

Reply:
that's my ugly mock :( i made the border thick because the rest of the component was so similar to actual UI that it didn't stand out as a placeholder (the way the issue linking prompt does)... any suggestions for which container to use?

Reply:
@Chrissy ^

Reply:
`<PanelBody/>` (without the header) might be a good place to start

https://storybook.getsentry.net/branches/master/index.html?selectedKind=UI%7CPanels&selectedStory=Basic%20Panel&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel